addons/storyshots/src/react/renderShallowTree.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Same here

Reply:
People who are already using Storyshots were already needed to install `react-test-renderer`, so for them, there won't be a problem. I need to add a section in Readme about these "optional peers", that people have to install by hand.
BTW, For angular and vue these deps are needed to configure jestconfig (I've already added it to readme)

Reply:
Makes sense to me